editor/editor_export.cpp

```diff
@@ -721,6 +724,7 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 	_edit_filter_list(paths, p_preset->get_include_filter(), false);
 	_edit_filter_list(paths, p_preset->get_exclude_filter(), true);
 
+    //PATCHES TAB
```

Comment:
Your indents don't match up - please fix.
Why is the comment in all caps?
Prefer a space after //